main.tf

```diff
@@ -57,6 +57,23 @@ resource "google_healthcare_fhir_store" "fhir_stores" {
       pubsub_topic = notification_config.value.pubsub_topic
     }
   }
+
+  dynamic "stream_configs" {
+    for_each = lookup(each.value, "stream_configs", [])
+
+    content {
+      resource_types = lookup(stream_configs.value, "resource_types", null)
```

Comment:
Can you also add `resource_types` in the example just for demonstration purpose?

Reply:
As discussed offline, there is also value in leaving resource_types out of the example as it tests that the "optional" aspect of the field is preserved.